src/components/DateRangeSelect.js

```diff
@@ -0,0 +1,116 @@
+import { useState, useEffect, useRef } from 'react'
+import Select from 'react-dropdown-select'
+import moment from 'moment'
+import { DateRange } from 'react-date-range';
+import 'react-date-range/dist/styles.css'; // main css file
+import 'react-date-range/dist/theme/default.css'; // theme css file
+
+export default function DateRangeSelect({ options, startDate, endDate, onChange }) {
+  const [selectedDateRangeOption, setSelectedDateRangeOption] = useState()
+  const [rangeState, setRangeState] = useState([
+    {
+      startDate: null,
+      endDate: null,
+      key: 'selection'
+    }
+  ]);
+
+  useEffect(() => {
+    setRangeState([
+      {
+        startDate: startDate,
+        endDate: endDate,
+        key: 'selection'
+      }
+    ])
+  }, [startDate, endDate])
+
+  const onSelectItem = (option) => {
+    if (option.id == 4) {
```

Comment:
what is this magic number? can we use constant? or better to have some property of option for this

Reply:
Yes we should be doing that `4` is the id for `All time` option

Reply:
https://github.com/gmx-io/gmx-stats/pull/9/files